src/Locales/it/Pode.psd1

```diff
@@ -0,0 +1,287 @@
+    nonEmptyScriptBlockRequiredForCustomLoggingExceptionMessage       = 'È richiesto uno ScriptBlock non vuoto per il metodo di registrazione personalizzato.'
+    encodingAttributeOnlyAppliesToMultipartExceptionMessage           = "L'attributo di codifica si applica solo ai corpi delle richieste multipart e application/x-www-form-urlencoded."
+    suppliedDateBeforeScheduleStartTimeExceptionMessage               = "La data fornita è precedente all'ora di inizio del programma a {0}"
+    unlockSecretRequiredExceptionMessage                              = "È necessaria una proprietà 'UnlockSecret' quando si utilizza Microsoft.PowerShell.SecretStore"
+    noLogicPassedForMethodRouteExceptionMessage                       = '[{0}] {1}: Nessuna logica passata.'
+    bodyParserAlreadyDefinedForContentTypeExceptionMessage            = 'Un body-parser è già definito per il tipo di contenuto {0}.'
+    invalidJwtSuppliedExceptionMessage                                = 'JWT fornito non valido.'
+    sessionsRequiredForFlashMessagesExceptionMessage                  = 'Le sessioni sono necessarie per utilizzare i messaggi Flash.'
+    semaphoreAlreadyExistsExceptionMessage                            = 'Un semaforo con il seguente nome esiste già: {0}'
+    invalidJwtHeaderAlgorithmSuppliedExceptionMessage                 = "Algoritmo dell'header JWT fornito non valido."
+    oauth2ProviderDoesNotSupportPasswordGrantTypeExceptionMessage     = "Il provider OAuth2 non supporta il tipo di concessione 'password' richiesto dall'utilizzo di un InnerScheme."
+    invalidAliasFoundExceptionMessage                                 = 'Alias {0} non valido trovato: {1}'
+    scheduleDoesNotExistExceptionMessage                              = "Il programma '{0}' non esiste."
+    accessMethodNotExistExceptionMessage                              = 'Il metodo di accesso non esiste: {0}'
+    oauth2ProviderDoesNotSupportCodeResponseTypeExceptionMessage      = "Il provider OAuth2 non supporta il tipo di risposta 'code'."
+    untestedPowerShellVersionWarningMessage                           = '[AVVERTENZA] Pode {0} non è stato testato su PowerShell {1}, poiché non era disponibile quando Pode è stato rilasciato.'
+    secretVaultAlreadyRegisteredAutoImportExceptionMessage            = "Una cassaforte segreta con il nome '{0}' è già stata registrata durante l'importazione automatica delle cassaforti segrete."
+    schemeRequiresValidScriptBlockExceptionMessage                    = "Lo schema fornito per il validatore di autenticazione '{0}' richiede uno ScriptBlock valido."
+    serverLoopingMessage                                              = 'Ciclo del server ogni {0} secondi'
+    certificateThumbprintsNameSupportedOnWindowsExceptionMessage      = 'Impronte digitali/nome del certificato supportati solo su Windows.'
+    sseConnectionNameRequiredExceptionMessage                         = "È richiesto un nome di connessione SSE, sia da -Name che da $`$WebEvent.Sse.Name"
```

Comment:
Typo in the escaping of "$" for "$WebEvent", there's an erroneous "$" in front of the backtick